rasterio/_base.pyx

```diff
@@ -9,6 +9,8 @@ import math
 import os
 import warnings
 
+import numpy
```

Comment:
I thought a goal of _base.pyx were numpy-free base classes?

Reply:
Yes, I think I recall that. We could move the method or even remove it since it's pretty trivial.